README.md

```diff
@@ -184,11 +182,20 @@ You can find a complete list of [all available rules here](docs/README.md).
 
 ## Available Configs
 
-This plugin exports a [`recommended` config](packages/plugin/src/configs/recommended.ts) that enforces good practices and an [`all` config](packages/plugin/src/configs/all.ts) that makes use of all rules (except for deprecated ones).
+<!-- prettier-ignore-start -->
+|Name|Description|
+|:-:|-|
+|[`schema-recommended`](packages/plugin/src/configs/schema-recommended.ts)|enables all recommended rules|
+|[`operations-recommended`](packages/plugin/src/configs/operations-recommended.ts) |enables all recommended rules|
+|[`schema-all`](packages/plugin/src/configs/schema-all.ts)|enables all rules (except for the rules that require `parserOptions.operations` option)|
+|[`operations-all`](packages/plugin/src/configs/operations-all.ts)|enables all rules|
```

Comment:
Does this include rules that require schema `parserOption`?

Reply:
Yes, this include all operations rules, that require (or not) `parserOptions.schema` option